test/studies/prk/stencil/PERFEXECOPTS

```diff
@@ -0,0 +1 @@
+--iterations=10 --order=1000
```

Comment:
I would be tempted to throw `--validate` and check for success in the .graph file